CHANGELOG.md

```diff
@@ -1,4 +1,88 @@
+* **New Resource:** `google_compute_instance_template_iam_member` ([#20954](https://github.com/hashicorp/terraform-provider-google/pull/20954))
+* **New Resource:** `google_compute_instance_template_iam_policy` ([#20954](https://github.com/hashicorp/terraform-provider-google/pull/20954))
+* **New Resource:** `google_gemini_code_repository_index` (GA) ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
+* **New Resource:** `google_gemini_repository_group` (GA) ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
+* **New Resource:** `google_gemini_repository_group_iam_member` (GA) ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
+* **New Resource:** `google_gemini_repository_group_iam_binding` (GA) ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
+* **New Resource:** `google_gemini_repository_group_iam_policy` (GA) ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
+* **New Resource:** `google_parameter_manager_parameter_version` ([#20992](https://github.com/hashicorp/terraform-provider-google/pull/20992))
+* **New Resource:** `google_redis_cluster_user_created_connections` ([#20977](https://github.com/hashicorp/terraform-provider-google/pull/20977))
+
+IMPROVEMENTS:
+* alloydb: added support for `skip_await_major_version_upgrade` field in `google_alloydb_cluster` resource, allowing for `major_version` to be updated ([#20923](https://github.com/hashicorp/terraform-provider-google/pull/20923))
+* apigee: added `properties` field to `google_apigee_environment` resource ([#20932](https://github.com/hashicorp/terraform-provider-google/pull/20932))
+* bug: added support for setting `custom_learned_route_priority` to 0 in 'google_compute_router_peer' by adding the `zero_custom_learned_route_priority` field ([#20952](https://github.com/hashicorp/terraform-provider-google/pull/20952))
+* cloudrunv2: added `build_config` to `google_cloud_run_v2_service` ([#20979](https://github.com/hashicorp/terraform-provider-google/pull/20979))
+* compute: added `pdp_scope` field to `google_compute_public_advertised_prefix` resource ([#20972](https://github.com/hashicorp/terraform-provider-google/pull/20972))
+* compute: adding `labels` field to resource_compute_interconnect_attachment ([#20971](https://github.com/hashicorp/terraform-provider-google/pull/20971))
+* compute: fixed a issue where `custom_learned_route_priority` was accidentally set to 0 during updates in 'google_compute_router_peer' ([#20952](https://github.com/hashicorp/terraform-provider-google/pull/20952))
+* filestore: added support for `tags` field to `google_filestore_instance` resource ([#20955](https://github.com/hashicorp/terraform-provider-google/pull/20955))
+* gemini: promoted resources `gemini_code_repository_index`, `gemini_repository_group` to GA. ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
```

Comment:
Is this just new resources? They're in the list already

Reply:
Oh, missed removing the line- I'd added the "New Resource" lines